DrinkApp.rb

```diff
@@ -1,43 +1,42 @@
 #!/usr/bin/env ruby
+load "./ui.rb"
 
 class DrinkApp
 drinkName = ""
```

Comment:
In Ruby syntax, methods with multiple words in them would look like this:

`drink_name`
`drink_description`
`wine_type` 

Something we didn't discuss during our one on one: If you have a set of methods that share `name` and `description`, what you really have is a class.

So, if you did something like `Drink.new(name, description)` , a drink class would be initialized with a name and a description passed into it.